### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by alexwidener** [rollbar/rollbar-agent#25]
I'd like to be able to run the daemon in Windows.

**Comment by brianr** [comment]
Hi @alexwidener - can you provide some more details about your use case? Are you hoping to use the agent to parse log files? Use it alongside rollbar-php? Something else?

**Comment by alexwidener** [comment]
Sure! 
Currently using pyrollbar to submit issues as they happen to rollbar, but it's causing the application to crash. Whenever the rollbar code is commented out, no crashes happen. In addition, whenever the application is being run through a debugger, the application does not suffer from crashes at all(with the rollbar code working). I ran 27 straight instances and no crashes at all. 

So, I want to be able to have rollbar-agent log what's happening to track down the information that's being transmitted and hopefully find what's causing the application to lock up. 

The application in question is Autodesk Maya 2015, SP5 which is running Python 2.7.3. 

I contacted Cory this morning and he was super helpful about switching the endpoint from https to http - which DOES improve results, but ultimately still crashes after a few attempts.

As of now, I believe the culprit might be the timeout, but having the agent would definitely be helpful in finding what's causing the lockup.
